Generator.Equals/EqualityTypeModelTransformer.cs

```diff
@@ -13,16 +11,19 @@ namespace Generator.Equals.Models;
 internal sealed class EqualityTypeModelTransformer
 {
     private readonly GeneratorAttributeSyntaxContext _context;
-    private readonly CancellationToken _token;
 
-    public EqualityTypeModelTransformer(GeneratorAttributeSyntaxContext context, CancellationToken token)
+    public EqualityTypeModelTransformer(GeneratorAttributeSyntaxContext context)
     {
         _context = context;
-        _token = token;
     }
 
-    public EqualityTypeModel? Transform()
+    public EqualityTypeModel? Transform(CancellationToken token = default)
```

Comment:
why pass in a `token` instead of doing `var token = context.CancellationToken;`?